contrib/python-gbinder/template.py

```diff
@@ -0,0 +1,24 @@
+pkgname = "python-gbinder"
+pkgver = "1.1.2"
+pkgrel = 0
+build_style = "python_pep517"
+hostmakedepends = [
+    "pkgconf",
+    "python-build",
+    "python-cython",
+    "python-installer",
+    "python-pytest",
+    "python-setuptools",
+]
+makedepends = [
+    "libgbinder-devel",
+    "python-devel",
+]
+pkgdesc = "Python bindings for libgbinder"
```

Comment:
you'll need

```
--- a/setup.py
+++ b/setup.py
@@ -10,7 +10,7 @@
         kw.setdefault(flag_map.get(token[:2]), []).append(token[2:])
     return kw
 
-USE_CYTHON = False
+USE_CYTHON = True
 if "--cython" in sys.argv:
     sys.argv.remove("--cython")
     USE_CYTHON = True
```

to build it since there's no way to pass an argv with pep517